hugo/layouts/papers/list-entry.html

```diff
@@ -3,16 +3,13 @@
 <p class="d-sm-flex align-items-stretch">
   <span class="d-block mr-2 text-nowrap list-button-row">
     {{- with $paper.url -}}
-    <a class="badge badge-primary align-middle mr-1" href="{{ . }}" data-toggle="tooltip" data-placement="top" title="Open PDF">
+    <a class="badge badge-primary align-middle mr-1" href="{{ . }}.pdf" data-toggle="tooltip" data-placement="top" title="Open PDF">
       pdf
     </a>
-      {{- if and (hasPrefix . $.Site.Params.baseURL) (eq . (strings.TrimSuffix ".pdf" .)) -}}
-    <a class="d-none" href="{{ . }}.pdf" title="Hidden link to PDF with extension">pdf</a>
-      {{- end -}}
     {{- end -}}
     {{- $bibfile := printf "/papers/%s/%s/%s.bib" (slicestr $volume_id 0 1) $volume_id .Params.anthology_id -}}
     {{- if (fileExists (printf "/data-export/%s" $bibfile)) -}}
-    <a class="badge badge-secondary align-middle mr-1" href="{{ $bibfile | relURL }}" data-toggle="tooltip" data-placement="top" title="Export to BibTeX">
+    <a class="badge badge-secondary align-middle mr-1" href="{{ $paper.url }}.bib" data-toggle="tooltip" data-placement="top" title="Export to BibTeX">
       bib
```

Comment:
The bib path is now `${ANTH_ID}.bib`, e.g., `https://aclweb.org/anthology/P19-1002.bib`, instead of the nested version.

Reply:
You're no longer using `$bibfile`, but the link is still within an if clause that checks for `$bibfile`.

Reply:
This is fine though, right? If the bibfile exists, we generate the appropriate link to it.

Reply:
I didn't change this because I'm not sure how to fix it. If the bibfile is present, we generate the canonical link to it, which is handled by a redirect internally.

Reply:
So, the question is whether we still want a check for the existence of the file in the Hugo template.

If we do, then this current solution is suboptimal IMO since it requires us manually remembering and updating the actual local path to the file (in the `$bibfile := ...` line), in case it ever changes.

If we do not, we should just throw out the surrounding lines altogether.